core/services/ardupilot_manager/main.py

```diff
@@ -23,29 +28,40 @@ def render(self, content: Any) -> bytes:
         ).encode(self.charset)
 
 
-app = FastAPI(default_response_class=PrettyJSONResponse)
+app = FastAPI(
+    title="ArduPilot Manager API",
+    description="ArduPilot Manager is responsible for managing ArduPilot devices connected to Companion.",
+    default_response_class=PrettyJSONResponse,
+)
 autopilot = ArduPilotManager()
 
 
 @app.get("/endpoints", response_model=List[Dict[str, Any]])
+@version(1, 0)
 def get_available_endpoints() -> Any:
     return [endpoint.asdict() for endpoint in autopilot.get_endpoints()]
 
 
 @app.post("/endpoints", status_code=status.HTTP_201_CREATED)
+@version(1, 0)
 def create_endpoints(response: Response, endpoints: Set[Endpoint] = Body(...)) -> Any:
```

Comment:
It should not accept endpoint with the same name:
![image](https://user-images.githubusercontent.com/1215497/117502853-821e3c80-af56-11eb-9e39-12a3ba054a0e.png)

Reply:
Yeah, agree.
Will add that check on the backend.